bumble/smp.py

```diff
@@ -457,9 +457,17 @@ class PairingDelegate:
     DISPLAY_OUTPUT_ONLY               = SMP_DISPLAY_ONLY_IO_CAPABILITY
     DISPLAY_OUTPUT_AND_YES_NO_INPUT   = SMP_DISPLAY_YES_NO_IO_CAPABILITY
     DISPLAY_OUTPUT_AND_KEYBOARD_INPUT = SMP_KEYBOARD_DISPLAY_IO_CAPABILITY
-
-    def __init__(self, io_capability = NO_OUTPUT_NO_INPUT):
+    DEFAULT_KEY_DISTRIBUTION          = (SMP_ENC_KEY_DISTRIBUTION_FLAG | SMP_ID_KEY_DISTRIBUTION_FLAG)
+
+    def __init__(
+        self,
+        io_capability=NO_OUTPUT_NO_INPUT,
+        local_i_key=DEFAULT_KEY_DISTRIBUTION,
```

Comment:
I suggest naming these variables `local_initiator_key_distribution` and `local_responder_key_distribution` to match the name used elsewhere in this class (also, without the `_distribution` suffix, one may get confused and think that this holds an actual key.

Reply:
Done